Directory.Build.props

```diff
@@ -40,10 +40,10 @@
   
   <!-- Platform detection -->
   <PropertyGroup>
-    <DefaultOSGroup Condition="'$(OS)'=='Unix' AND Exists('/Applications')">OSX</DefaultOSGroup>
+    <DefaultOSGroup Condition="'$(OS)'=='Unix' AND $([MSBuild]::IsOSPlatform('OSX'))">OSX</DefaultOSGroup>
```

Comment:
Do we need both checks or is the latter sufficient? Same question for the changei n L46.

Reply:
I think the latter one is sufficient for all operating systems (L43-L46). Updated in a7f45836eefe and also reverted the change `unset OS` from `build.sh`. :)